src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/IncrementalBuildTest.cs

```diff
@@ -12,6 +13,29 @@ namespace Xamarin.Android.Build.Tests
 	[Parallelizable (ParallelScope.Children)]
 	public class IncrementalBuildTest : BaseTest
 	{
+		[Test]
+		public void CheckNothingIsDeletedByIncrementalClean ([Values (true, false)] bool enableMultiDex)
+		{
+			// do a release build
+			// change one of the properties (say AotAssemblies) 
+			// do another build. it should NOT hose the resource directory.
+			var path = Path.Combine ( "temp", TestName );
+			var proj = new XamarinAndroidApplicationProject () {
```

Comment:
Whoops I think this one should be the new `XamarinFormsAndroidApplicationProject`.